cypress/e2e/hub/api-tokens.cy.ts

```diff
@@ -1,11 +1,22 @@
 import { APITokens } from './constants';
 
 describe('API Tokens', () => {
-  before(() => {
+  beforeEach(() => {
     cy.hubLogin();
   });
```

Comment:
Is there a particular reason to login before every test? If we can re-use the session that would make the tests a bit faster.

Reply:
This may not be needed. I'm debugging on GH so it's still WIP. I want to get it green before I try to remove it :)

Reply:
And it wasn't. Nice catch!